lib/components/ConflictPicker.vue

```diff
@@ -372,7 +373,7 @@ export default Vue.extend({
 		position: sticky;
 		z-index: 10;
 		top: 0;
-		padding: var(--margin);
+		padding: 0 var(--margin);
```

Comment:
lessen the padding for the header, as without this, the header has more padding than previously and looks really bloated